core/chains/evm/gas/arbitrum_estimator.go

```diff
@@ -110,7 +107,6 @@ func (a *arbitrumEstimator) GetLegacyGas(ctx context.Context, calldata []byte, l
 	if err != nil {
 		return
 	}
-	gasPrice = a.gasPriceWithBuffer(gasPrice, maxGasPriceWei)
```

Comment:
Since we reverted the Broadcaster changes, we may want to add `gasPriceWithBuffer` back to the arb estimator